apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -402,6 +422,37 @@ export class DotEditContentHtmlService {
         this.dotEditContentToolbarHtmlService.addContainerToolbar(doc);
     }
 
+    private injectInlineEditingScripts(): void {
+        const doc = this.getEditPageDocument();
+        const TINYMCE = `/html/js/tinymce/js/tinymce/tinymce.min.js`;
+        const tinyMceScript: HTMLScriptElement = this.dotDOMHtmlUtilService.creatExternalScriptElement(
+            TINYMCE
+        );
+
+        this.initInlineEditing();
+        doc.body.append(tinyMceScript);
+    }
+
+    private initInlineEditing(): void {
+        const doc = this.getEditPageDocument();
+        const tinyMceInit: HTMLScriptElement = this.dotDOMHtmlUtilService.createInlineScriptElement(
+            INLINE_TINYMCE_SCRIPTS
+        );
+
+        this.dotLicenseService
+            .isEnterprise()
+            .pipe(take(1))
+            .subscribe((isEnterprise) => {
```

Comment:
type and since you are using a `if` you might want to use the `filter` operator here.